<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminModeDialog</class>
 <widget class="QDialog" name="AdminModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>301</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>301</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AdminMode</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tiremanagement.qrc">
    <normaloff>:/img/administrator.png</normaloff>:/img/administrator.png</iconset>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>301</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Modify">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_New">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>140</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>LoginInfo</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_UserID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UserID</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Find">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Return">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_UserID</tabstop>
  <tabstop>lineEdit_Password</tabstop>
  <tabstop>pushButton_New</tabstop>
  <tabstop>pushButton_Modify</tabstop>
  <tabstop>pushButton_Delete</tabstop>
  <tabstop>pushButton_Find</tabstop>
  <tabstop>pushButton_Return</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources>
  <include location="tiremanagement.qrc"/>
 </resources>
 <connections/>
</ui>
